snippets/collapse.md

```diff
@@ -0,0 +1,25 @@
+---
+title:  collapse
+tags: array,beginner
+---
+
+Collapse a list of arrays into a single.
+
+Use `array_reduce()` to reduce each item into a single array, if the item
+is a array, merge the array with final item, if not, adds the item in a array and merge too.
+
+```php
+function collapse($array) 
+{
```

Comment:
Please follow the snippet template's formatting for code. Indentation should be 2 extra spaces per indentation level.